### PR TITLE
WIP: Adding CucuShift tests to ci-operator

### PR DESCRIPTION
**Pull request by jhou1**
This is a first attempt to add QE's [cucushift](https://github.com/openshift/verification-tests) tests to the ci-operator following the instructions in https://docs.ci.openshift.org/docs/. 
Please kindly let me know if I had been complying with the ci-operator's practice in creating jobs and tests.

cc @mffiedler @liangxia @jianlinliu

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 1]
```diff
@@ -23738,6 +23738,81 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```
Comment (jianlinliu) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:23741]: Anyone can point me how this prowjob hooks ci-operator/step-registry/cucushift/cucushift-workflow.yaml

File: ci-operator/step-registry/cucushift/cucushift-commands.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# prepare oc client
+mkdir /tmp/bin
+export PATH=$PATH:/tmp/bin
+curl https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz | tar xvzf - -C /tmp/bin/ oc
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/cucushift/cucushift-commands.sh:10]: Please inject the CLI using this doc: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

So on the  `ci-operator/step-registry/cucushift/cucushift-ref.yaml` use `cli:latest` to get the correct `oc` for the version of OCP being tested

**Review by stevekuznetsov — COMMENTED** [review]
This looks reasonable to me

**Comment by stevekuznetsov** [comment]
If your Cucumber suite can create jUnit output and put it into `$ARTIFACT_DIR` it will be displayed as failing test cases in the job UI.

**Comment by jhou1** [comment]
Cool, I'll put the jUnit results there, thank you!

**Comment by liangxia** [comment]
The error indicates a server flake.
```
ERRO[2021-08-17T02:55:41Z] Some steps failed:                           
ERRO[2021-08-17T02:55:41Z] 
  * failed to generate steps from config: failed to resolve release initial: failed to request latest release: server responded with 503: <html>
```

/test pj-rehearse

**Comment by jhou1** [comment]
This is a sample for exploring how we should organize our tests in Prow. Closing now and will create PRs for dedicated profiles.
/close

**Pull request closed without merging**